src/style.css

```diff
@@ -2,2 +1,76 @@
+* {
   margin: 0;
+  list-style: none;
+  text-decoration: none;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url('./fonts/Roboto-Medium.ttf') format('truetype');
```

Comment:
Ensure that the font file path is correct and that the font is loaded properly. If the font is not displaying as expected, check the path and file permissions.